blog/2024/04-28-jellyfin-release-10.9.0/index.mdx

```diff
@@ -0,0 +1,124 @@
+
+* Parental ratings are significantly improved, with better enforcement, inheritable ratings, and more.
+
+* LiveTV and Collection permissions are now discrete and configurable per-user.
+
+* All playback endpoints now enforce authentication, preventing anonymous users from streaming by potentially guessing item IDs (highly unlikely, but not impossible).
+
+* The EasyPassword feature has been removed as this was a big security risk especially for administrator accounts; PIN login is still supported.
+
+* User permission handling has been unified and numerous bugs fixed, ensuring a more secure server from untrusted users.
+
+### Core Server & Networking
+
+* The underlying .NET framework has been upgraded to version 8, the latest major and LTS release. This brings us many under-the-hood improvements and changes to help with our ongoing effort to streamline the codebase and improve overall performance.
+
+* Our minimum FFmpeg version has been bumped from 4.0 to 4.4 to take advantage of many new features. For users of our Jellyfin FFmpeg build (Docker, `.deb`/Debuntu repo, and Windows installs), you're already on an even newer version now with FFmpeg 6.0.1, but this minimum change would affect any other platforms.
+
+* The server now supports in-process restarting, and removes the old hacky `restart.sh` method. This should make things like installing plugins much more robust.
+
+* Uploaded images are now subject to much more verification, helping plug several security holes in this feature.
+
+* The backend SQLite database now supports connection pooling, which should improve performance for database operations.
+
+* Support for the HappyEyeballs engine improving dual-stack IPv4/IPV6 network support.
+
+* Improved handling of IP binding to fully respect these settings (including DLNA and SSDP).
+
+### Packaging
+
+* The `jellyfin` service user is now automatically added to the `render` and `video` groups on Debian/Ubuntu systems for new installs, helping improve the onboarding for hardware acceleration.
+
+* Packaging is now unified with a new CI system, as outlined in [our previous blog post](https://jellyfin.org/posts/new-ci-new-repo). As a normal user you should see no change here, but this helps massively improve the build process, and if you are building your own packages please reference [the new packaging repository](https://github.com/jellyfin/jellyfin-packaging) for the updated process.
+
+### Transcoding & FFmpeg
+
+* Several audio improvements: support for Opus, FLAC, and ALAC codecs in the HLS engine; improved support and handling for transcoding DTS and TrueHD audio streams; new codec enforcement; better bitrate calculations; new audio normalization features; and stereo downmixing algorithm selection.
+
+* FFmpeg segments are now automatically deleted during transcoding, reducing the space of the transcoding temporary directory.
+
+* Support for direct playback of DVD and BluRay data folders, preferred over ISOs.
```

Comment:
Should be `direct stream` - since it is a folder structure, direct play will never be possible

Reply:
Changed to "direct stream playback" to avoid implying "direct play".